From mdharm-usb@one-eyed-alien.net  Wed Nov 12 11:30:16 2008
From: Paul Bolle <pebolle@tiscali.nl>
Date: Thu, 30 Oct 2008 08:42:43 -0700
Subject: USB: storage: adjust comment in Kconfig
To: Linux USB <linux-usb@vger.kernel.org>, Greg KH <greg@kroah.com>, USB Storage List <usb-storage@lists.one-eyed-alien.net>
Message-ID: <20081030154243.GA16166@one-eyed-alien.net>
Content-Disposition: inline


From: Paul Bolle <pebolle@tiscali.nl>

Since commit 65934a9 ("Make USB storage depend on SCSI rather than selecting
it [try #6]") the comment at the top of drivers/usb/storage/Kconfig is
incorrect. Adjust it to the current situation.

Signed-off-by: Paul Bolle <pebolle@tiscali.nl>
Signed-off-by: Matthew Dharm <mdharm-usb@one-eyed-alien.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/storage/Kconfig |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/drivers/usb/storage/Kconfig
+++ b/drivers/usb/storage/Kconfig
@@ -2,8 +2,8 @@
 # USB Storage driver configuration
 #
 
-comment "NOTE: USB_STORAGE enables SCSI, and 'SCSI disk support'"
-comment "may also be needed; see USB_STORAGE Help for more information"
+comment "NOTE: USB_STORAGE depends on SCSI but BLK_DEV_SD may also be needed;"
+comment "see USB_STORAGE Help for more information"
 	depends on USB
 
 config USB_STORAGE
